tidy3d/components/tcad/doping.py

```diff
@@ -20,12 +20,14 @@ def _normal_dim(self):
         """Returns the normal direction if the box is 2D. False otherwise"""
```

Comment:
It seems to returns *some* normal_dim when the box is 2D or 3D, and it raises otherwise.